xsd/jps_ini_core.xsd

```diff
@@ -428,7 +428,6 @@
                   <xs:attribute name="description" use="required">
                     <xs:simpleType>
                       <xs:restriction base="xs:string">
-                        <xs:enumeration value="gompertz" />
```

Comment:
Isn’t the velocity model missing?

Reply:
The whole xsd needs an update I think. Just wanted to remove gompertz...